src/algebra/big_operators/multiset.lean

```diff
@@ -49,6 +49,11 @@ begin
   rw coe_prod,
 end
 
+@[to_additive]
+lemma to_list_map_prod {ι : Type*} (s : multiset ι) (f : ι → α) :
+  (s.to_list.map f).prod = (s.map f).prod :=
+by rw [← coe_prod, ← coe_map, s.coe_to_list]
+
```

Comment:
This probably isn't worth having any more; if you really want to go this way, I'd expect separate `to_list_map` and `to_list_prod` lemmas.

Reply:
I agree: I had only added it to get the previous proof going, but I removed it now.